#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_httpdconf.dpatch by  <seneca@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changing paths in Apache config

@DPATCH@

diff -Naur ledgersmb-1.1.99rc5-new/ledgersmb-httpd.conf ledgersmb-1.1.99rc5-dirty/ledgersmb-httpd.conf
--- ledgersmb-1.1.99rc5-new/ledgersmb-httpd.conf	2007-02-23 00:44:50.000000000 -0500
+++ ledgersmb-1.1.99rc5-dirty/ledgersmb-httpd.conf	2007-02-19 20:46:47.000000000 -0500
@@ -1,7 +1,7 @@
 
-Alias /ledgersmb WORKING_DIR
+Alias /ledgersmb /usr/share/ledgersmb/
 
-<Directory WORKING_DIR>
+<Directory /usr/share/ledgersmb>
   AllowOverride All
   AddHandler cgi-script .pl
   Options ExecCGI Includes FollowSymlinks
@@ -13,27 +13,12 @@
   </Files>
 </Directory>
 
-<Directory WORKING_DIR/users>
+<Directory /usr/share/ledgersmb/bin>
   Order Deny,Allow
   Deny from All
 </Directory>
   
-<Directory WORKING_DIR/bin>
-  Order Deny,Allow
-  Deny from All
-</Directory>
-  
-<Directory WORKING_DIR/utils>
-  Order Deny,Allow
-  Deny from All
-</Directory>
-  
-<Directory WORKING_DIR/spool>
-  Order Deny,Allow
-  Deny from All
-</Directory>
-
-<Directory WORKING_DIR/LedgerSMB>
+<Directory /usr/share/ledgersmb/LedgerSMB>
   Order Deny,Allow
   Deny from All
 </Directory>
